pillar.example

```diff
@@ -48,3 +48,7 @@ logstash_forwarder:
             YRQ2WF1aPoB1CccqG0zn9thKJI0yhJUZ8xeY+KJx+/3/H6AckEduJZdNdWiaUi6p
             NHQnOJJ/BYleQ0Q=
             -----END CERTIFICATE-----
+	  repo:
```

Comment:
Doesn't this need the ":lookup:" key as they are being merged into the os_family_map in `map.jinja` ?